### ISSUE TITLE
Auto save on leaving app

### ISSUE THREAD

**Issue by smaugilein** [stefan-niedermann/nextcloud-deck#747]
Please use **[GitHub reactions](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/)** 👍 to show that you are affected by the same issue. Please don't comment if you have no relevant information to add!

**Is your feature request related to a problem? Please describe.**

To use nextcloud Deck as a notes app I am missing an important feature: Often I want to quickly add an item to a note (list) and forget to click on the checkmark (confirmation) after editing, but switch to the next app and the changes are not saved.
If I now access the list via my widget on the homescreen (shows this one list), the old content is loaded.

**Describe the solution you'd like**

Automatic saving of lists after leaving app or pressing back.

**Comment by stefan-niedermann** [comment]
Yeah, i get the problem. We decided explicitely to not autosave but use the more explicit ✔️ to save, but we will rethink about it. Maybe we can make a setting for this or so.

**Comment by smaugilein** [comment]
I prefer that solution. Thanks a lot!
